python/.gitignore

```diff
@@ -151,4 +151,4 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
+#.idea/
```

Comment:
This file should not be changing as part of this PR

Reply:
Our plan is to add the changes to a new branch called feat 4122 text to speech branch, it would fix this I assume.